jobs/aws/eks-distro/kops-build-postsubmit.yaml

```diff
@@ -0,0 +1,46 @@
+# Copyright 2020 Amazon.com Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+postsubmits:
+  aws/eks-distro:
+  - name: kops-build-postsubmit
```

Comment:
I'm not sure what case where `run_if_changed: ".*"` and `always_run: false` would be false.

Reply:
Nothing comes to my mind at this point.  Right now they are essentially the same thing. Probably we can change the search pattern later to something more restrictive  .

Reply:
does `always_run` not filter on specific branches?

Reply:
It looks like `always_run` is just a boolean indicating if you are going run all the time or filter. Since we really only merge to main right now, we don't need to filter on branch at the moment.

Reply:
I tried to run against a branch which is not main with this always_run flag set and the run_if_changed part commented out. And it did trigger a job.

Reply:
That is what I would expect and I think that is fine. If it because a problem, we can do something about it.

Reply:
Does this job wait until the other jobs finish for postsubmit?

Reply:
I don't think it does.